lib/plugins/aws/lib/naming.js

```diff
@@ -504,4 +498,18 @@ module.exports = {
   getCustomResourceEventBridgeResourceLogicalId(functionName, idx) {
     return `${this.getNormalizedFunctionName(functionName)}CustomEventBridge${idx}`;
   },
+  // API Gateway Account Logs Write Role
+  getCustomResourceApiGatewayAccountCloudWatchRoleHandlerFunctionName() {
+    return 'custom-resource-apigw-account-cw-role';
```

Comment:
Should we make this shorter to give more room for service + stage name?

Do you know what our current effective limit is for service + stage combo, given other resources we create? We should try to not shorter than any further.

Reply:
I stripped the `account` to make it safer